api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/PersonTranslatorImpl.java

```diff
@@ -69,6 +72,23 @@
 		person.setActive(!openmrsPerson.getVoided());
 		person.setBirthDate(openmrsPerson.getBirthdate());
 		
+		if (openmrsPerson.getBirthdateEstimated() != null) {
+			if (openmrsPerson.getBirthdateEstimated()) {
+				DateType dateType = new DateType();
+				Calendar calendar = Calendar.getInstance();
```

Comment:
Instead of using `Calendar` can we change this to use the `java.time` package classes?

Reply:
@ibacher thank. i will change this.

Reply:
changed the calendar to java.time package classes.